user_manual/files/deleted_file_management.rst

```diff
@@ -53,7 +52,8 @@ by setting the ``trashbin_retention_obligation`` value. Files older than the
 ``trashbin_retention_obligation`` value will be deleted permanently.
 Additionally, Nextcloud calculates the maximum available space every time a new
 file is added. If the deleted files exceed the new maximum allowed space
-Nextcloud will expire old deleted files until the limit is met once again.
+Nextcloud will permanently delete old removed files which will expire until the
+limit is met once again.
 
 .. note:: Your administrator may have configured the trash bin retention period 
-   to override the storage space management. See `admin documentation <https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#deleted-items-trash-bin>`_ for more details.
\ No newline at end of file
+   to override the storage space management. See `admin documentation <https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#deleted-items-trash-bin>`_ for more details.
```

Comment:
What am I missing here? I just can't find the change.

Reply:
There are no changes here. The sentence was shifted on the line of code because the previous sentence was extended by one line more.